common/springUtilities/teamFunctions.lua

```diff
@@ -9,6 +9,7 @@ local function getSettings()
 
 	local allyTeamCount, playerCount = 0, 0
 	local isSinglePlayer, is1v1, isTeams, isBigTeams, isSmallTeams, isRaptors, isScavengers, isPvE, isCoop, isFFA, isSandbox = false, false, false, false, false, false, false, false, false, false, false
+	local ScavTeamID, ScavAllyTeamID = nil, nil
```

Comment:
```suggestion
	local scavTeamID, scavAllyTeamID = nil, nil
```
Locals start with lowercase